.circleci/config.yml

```diff
@@ -33,7 +33,8 @@ defaults: &defaults
           pip install --upgrade setuptools
           python setup.py load_highcharts --highcharts=node_modules/highcharts/,node_modules/highcharts/modules
           python setup.py load_wkhtmltopdf --wkhtmltopdf=wkhtmltox/bin
-          pip install .[docs_and_notebook,plotly,bokeh,tests,server]
+          pip install .[docs_and_notebook,plotly,bokeh,tests,server,code]
+          pip install Pygments==2.16.1
```

Comment:
the pin shouldn't be required?

Reply:
As mentioned in the commit message this is so regression tests don't fail due to new releasees of pygments.